test/Options.h

```diff
@@ -14,17 +14,18 @@
 	You should have received a copy of the GNU General Public License
 	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
 */
-/** @file TestHelper.h
+/** @file Options.h
```

Comment:
I'm not sure, if we still need this annotation. I mean, I don't know if we work towards a proper Doxygen documentation or not?